frontend/src/hooks/useChartMutable.ts

```diff
@@ -1,9 +1,8 @@
 import { PANEL_TYPES } from 'constants/queryBuilder';
+import { PanelTypeAndGraphManagerVisibility } from 'container/GridGraphLayout/Graph/FullView/contants';
```

Comment:
can we take this as props probably by this we can handle various places

Reply:
Okay